benchmark/utils.py

```diff
@@ -70,7 +66,7 @@ def init_model(args):
                      lr=choice(lr),
                      epoch=epoch,
                      gpu=gpu,
-                     alpha=choice(alpha),
+                     wight=choice(alpha),
```

Comment:
It should be weight instead of wight. Similar issue for the follwing changes.

Reply:
yes, it was a typo mistake. I will change it ASAP.

Reply:
done!